lib/ClaimVerifier.ts

```diff
@@ -33,44 +29,38 @@ export class ClaimVerifier {
   }
 
   /**
-   * @description Checks that claim represents role and was
+   * @description Checks that credential represents role and was
    * issued by issuer required by role definition
    *
-   * @param claim: role claim
+   * @param credential: Role EIP191Jwt
    */
-  private async verifyRole(claim: Required<OffchainClaim>): Promise<{
-    name: IRoleDefinition['roleName'];
-    namespace: OffchainClaim['claimType'];
+  private async verifyRole(credential: Required<RoleEIP191JWT>): Promise<{
+    name: IRoleDefinitionV2['roleName'];
+    namespace: RoleEIP191JWT['payload']['claimData']['claimType'];
   } | null> {
     // if (!(await this.verifySignature(claim))) {
     //   return null;
     // }
 
-    const role = await this.getRoleDefinition(claim.claimType);
-    if (!role) {
-      return null;
-    }
-
-    if (role.version !== claim.claimTypeVersion) {
-      return null;
-    }
+    if (credential.payload) {
```

Comment:
Personally, I think the code is easier to read if the `if` statement is inverted and we handle the negative condition immediately instead of needing to scan down to see what happens if this isn't true
```suggestion
    if (!credential?.payload) {
      // Maybe add some log here
      return null;
    }
```